content/en/docs/contribute/style/diagram-guide.md

```diff
@@ -0,0 +1,577 @@
+{{< /note >}}
+
+Proper captioning involves three components: diagram, diagram caption and the diagram referral. As an example, consider a diagram `.svg` image file saved to `/images/docs/components-of-kubernetes.svg`.
+
+**Diagram**
+
+Here is the `{{</* figure */>}}` shortcode for the diagram:
+```text
+{{</* figure src="/images/docs/components-of-kubernetes.svg" alt="Kubernetes pod running inside a cluster" class="diagram-large" */>}}
+```
+You should only pass the `src`, `alt` and `class` values into the `{{</* figure */>}}` shortcode. You can adjust the size of the diagram using `diagram-large`, `diagram-medium` and `diagram-small` classes. You can't style the `caption` value in this shortcode, and it only supports diagram image files. 
+
+{{< note >}}
+Diagrams created using the `Inline` method don't use the `{{</* figure */>}}` shortcode.
+{{< /note >}}
+
+**Diagram Caption**
+
+Next, add a diagram caption. You must use the following format:
+```text
+<figcaption><p style="padding-left: 310px; font-weight: bold">Figure #. CAPTION TEXT</p></figcaption><br>
+```
+Here are several items to consider when adding diagram captions: 
+
+* Use`<figcation></figcaption>` tags to add a diagram caption for `Inline`, `Mermaid+SVG` and `External Tool` diagrams. 
+* Wrap caption text inside `<p></p>` tags using `style="padding-left: 310px; font-weight: bold"`. This provides a consistent style for the caption text.
+* Prepend your diagram caption with `Figure NUMBER.`. You must use `Figure` and the number must be unique for each diagram in your documentation page. 
+* Add your diagram caption text after the `Figure NUMBER.` on the same line. Keep it short.
+* Position your diagram caption __BELOW__ your diagram.
+* You can hyperlink your caption using `<a></a>` tags.
+
+Here is the caption for the `/images/docs/components-of-kubernetes.svg` diagram: 
+```text
+<figcaption><p style="padding-left: 310px; font-weight: bold">Figure 4. Kubernetes Architecture.</p></figcaption><br>
+```
+Here is the hyperlinked caption for the same diagram:
+```text
+<figcaption><a href="https://kubernetes.io/docs/concepts/overview/components/"><p style="padding-left: 310px; font-weight: bold">Figure 4. Kubernetes Architecture Test.</p></a></figcaption><br>
+```
+
+Check out the source code for this page, `diagram-guide.md`, for more examples of diagram captions.
+
+**Diagram Referral**
+
+Finally, you can add a diagram caption referral. This is used inside your text. It allows a user to connect your text with the diagram. The `Figure NUMBER` in your referral and caption must match.   
+
+Here is an example of a caption referral included in your content text:
+```text
+Figure 9 depicts the components of the Kubernetes architecture. The control plane ...
+```
+Diagram referrals are optional and there are cases where they might not be suitable. If you are not sure, add a diagram referral to your text to see if it looks and sounds okay. When in doubt, use a diagram referral.
```

Comment:
We can use hypertext to link to diagrams, rather than numbered figures. Let's omit this and if we find we need these, add a hypertext mechanism instead.